abnf/odata-aggregation-abnf.txt

```diff
@@ -109,8 +112,8 @@ aggregateTrafo  = %s"aggregate" OPEN BWS aggregateItem *( BWS COMMA BWS aggregat
 aggregateItem   = %s"$count" asAlias 
                 / aggregateExpr
 aggregateExpr   = commonExpr aggregateWith [ aggregateFrom ] asAlias
-                / pathPrefix primitiveProperty aggregateWith [ aggregateFrom ] asAlias
-                / pathPrefix customAggregate [ customFrom asAlias ]
+                / [ aggPathPrefix "/" ] ( primitiveProperty / primitiveColProperty ) aggregateWith [ aggregateFrom ] asAlias

```

Comment:
Is aggregation of primitive instance annotations allowed?

Reply:
Why not. Just in case anyone gets that deep into instance annotations _and_ aggregation 🤣